lib/wibox/widget/slider.lua

```diff
@@ -297,6 +334,17 @@ function slider:draw(_, cr, width, height)
 
     bar_shape(cr, width - x_offset - right_margin, bar_height or height)
 
+    if bar_active_color and type(bar_color) == "string" and type(bar_active_color) == "string" then
+        local bar_active_width = active_rate * (width - x_offset - right_margin)
+            - (handle_width - handle_border_width/2) * (active_rate - 0.5)
+        cr:set_source(color.create_pattern{
+            type        = "linear",
+            from        = {0,0,0},
```

Comment:
Linear patterns only require `x` and `y`, no radius.

```suggestion
            from        = {0, 0},
```